web/src/invite.js

```diff
@@ -267,7 +279,28 @@ function open_invite_user_modal(e) {
         show_select_default_streams_option: stream_data.get_default_stream_ids().length !== 0,
     });
 
+    function hide_invite_user_modal() {
+        overlays.close_active_modal();
+    }
+
     function invite_user_modal_post_render() {
+        const pill_config = {
+            do_not_show_user_group_size: true,
+        };
+        $pill_container = $(`#invite-user-form .pill-container`).expectOne();
+        pills = user_group_pill.create_pills($pill_container, pill_config);
+        const $input = $pill_container.children(".input");
+        pill_typeahead.set_up($input, pills, {
+            user_group: true,
+            helpOnEmptyStrings: true,
```

Comment:
We generally do not use camel-case. I know bootstrap uses it but we can still just pass "help_on_empty_strings" here and do not need to use what bootstrap uses here.